morph_tool/amira_converter.py

```diff
@@ -0,0 +1,172 @@
+"""Amira morphology format loader.
+
+This package is adaped from
+
+https://github.com/zibneuro/udvary-et-al-2022/blob/master/structural_model/util_amira.py
+
+to only load amira files into a MorphIO morphology.
+
+WARNING: this is not an official amira loader, it may contain bugs or unsupported features.
+"""
+from collections import defaultdict
+import numpy as np
+import pandas as pd
+
+from morphio import PointLevel, SectionType
+from morphio.mut import Morphology
+
+# pylint: disable=too-many-locals
+
+
+def get_section_data(lines):
+    """Read file and extract data per section into a dict of lists."""
+    _converter = {
+        "@1": "node_points",
+        "@2": "node_label",
+        "@3": "edge",
+        "@4": "n_points",
+        "@5": "label",
+        "@6": "point",
+        "@7": "radius",
+    }
+    sections = defaultdict(list)
+    current_section = ""
+    for line in lines:
+        if line.startswith("@"):
+            current_section = line
+        elif not line:
+            current_section = ""
+        elif current_section:
+            sections[_converter[current_section]].append(line)
+    return sections
```

Comment:
```suggestion
    return dict(sections)
```
It's usually better not to carry around the default dict around, given that it is not needed anymore.